### DISCUSSION TITLE
How to set filter `open` as the default value in my own discussion

### DISCUSSION THREAD

**Discussion by hooke007** [community/community#128865] [Discussions]
### Select Topic Area

Bug

### Body

I noticed it was started days ago that everytime I go to any repo's discussion page. The filter is always `all`. I do not want see those deprecated/dup posts, because that is why they were cloesd.

**Comment by hooke007** [comment]
@community @github-staff And why your discussion is different? 
![image](https://github.com/community/community/assets/41094733/8676c0c8-cf96-418d-b898-6321f2b4c5da)

**Reply by LiteBrite82** [reply]
Hi @hooke007, 

I'm not sure what you mean; I think everyone has the same filter options in the dropdown. Do you mean what filter the page loads by default?

**Reply by hooke007** [reply]
yes

**Reply by LiteBrite82** [reply]
Yeah, I see it now... I never noticed. Not sure if its a feature or a bug....

**Comment by chodid** [comment]
Any news on whether it's a bug or a feature? 
It'd be awesome to roll it back to a default of "open" or give the opportunity to manually select the default filter.

**Reply by hooke007** [reply]
I reported to others too. But no staff knew it had changed.
So it's a bug.

**Reply by chodid** [reply]
Thanks for the reply!
Did you get any feedback, whether they put it onto their agenda?

**Reply by hooke007** [reply]
![image](https://github.com/user-attachments/assets/35e7bd08-f40f-44cf-96cf-d208b82c667d)
I have no idea. It was cloesd weeks ago.

**Comment by clizbe** [comment]
Agree it would be nice to have the default as open. Should be an option for maintainers.
